lib/commands/update.js

```diff
@@ -58,6 +58,11 @@ class UpdateCommand extends Command {
         return instance.running().then((isRunning) => {
             if (isRunning) {
                 instance.loadRunningEnvironment(true);
+
+                // argv.restart will only be undefined if it wasn't passed in
+                if (argv.restart === undefined) {
```

Comment:
might be better (since we've upgraded to yargs 12 which enables this), to use the `in` operator:
```suggestion
                if (!('restart' in argv)) {
```
imo it's a _little_ cleaner